armory/art_experimental/attacks/poison_loader_dlbd.py

```diff
@@ -33,11 +33,26 @@ def mod(x):
         size = tuple(size)
         mode = kwargs.get("mode", "RGB")
         blend = kwargs.get("blend", 0.6)
-        base_img_size_x = kwargs.get("base_img_size_x", 48)
-        base_img_size_y = kwargs.get("base_img_size_y", 48)
+
         channels_first = kwargs.get("channels_first", False)
-        x_shift = kwargs.get("x_shift", (base_img_size_x - size[0]) // 2)
-        y_shift = kwargs.get("y_shift", (base_img_size_y - size[1]) // 2)
+
+        random_placement = kwargs.get("random", False)
+        if random_placement:
+            x_shift = 0
+            y_shift = 0
+        elif "x_shift" in kwargs and "y_shift" in kwargs:
+            x_shift = kwargs.get("x_shift")
+            y_shift = kwargs.get("y_shift")
+        elif "base_img_size_x" in kwargs and "base_img_size_y" in kwargs:
+            # default to center of image
+            base_img_size_x = kwargs.get("base_img_size_x")
+            base_img_size_y = kwargs.get("base_img_size_y")
+            x_shift = (base_img_size_x - size[0]) // 2
+            y_shift = (base_img_size_y - size[1]) // 2
```

Comment:
Are these kwargs documented anywhere? I can't seem to find what they are doing although I can make an educated guess. Might be worth adding something to https://github.com/twosixlabs/armory/blob/develop/docs/poisoning.md#configuration-files about what kwargs are available.

Reply:
They are the kwargs for the art class, so the documentation would be there in ART.  I'll add a note in the doc